openmoltools/openeye.py

```diff
@@ -62,13 +62,18 @@ def get_charges(molecule, max_confs=800, strictStereo=True, normalize=True, keep
 
     charged_copy = generate_conformers(molecule, max_confs=max_confs, strictStereo=strictStereo)  # Generate up to max_confs conformers
 
-    status = oequacpac.OEAssignPartialCharges(charged_copy, oequacpac.OECharges_AM1BCCSym)  # AM1BCCSym recommended by Chris Bayly to KAB+JDC, Oct. 20 2014.
+    # 2017.2.1 OEToolkits new charging function
+    status = oequacpac.OEAssignCharges(charged_copy, oequacpac.OEAM1BCCCharges())
 
     if not status:
-        raise(RuntimeError("OEAssignPartialCharges returned error code %d" % status))
+        # OEAssignCharges returns False if failure.
+        raise(RuntimeError("OEAssignCharges failed."))
 
     #Determine conformations to return
     if keep_confs == None:
+        logger.warning(
+            "keep_confs was set to None. Returned molecules will not be charged.")
```

Comment:
@nathanmlim - Andrea is right, unless you've changed it the warning message here is wrong. The behavior we'd implemented (as per the doc string) was that if `keep_confs = None` then the computed charges are applied to the provided conformation of the molecule, rather than any of the generated conformations.